### PR TITLE
PE-147: [wmi-config.xml] - migrate config file to new CM

### PR DESCRIPTION
**Pull request by freddy61025**
https://issues.opennms.org/browse/PE-147

### CODE REVIEW

File: features/config/mock/src/main/java/org/opennms/netmgt/dao/mock/ConfigurationManagerServiceMock.java [hunk 1]
```diff
@@ -113,6 +114,9 @@ public void changeConfigDefinition(String configName, ConfigDefinition configDef
         } else if ("notifd".equals(configName)) {
             def = XsdHelper.buildConfigDefinition("notifd", "notifd-configuration.xsd",
                     "notifd-configuration", ConfigurationManagerService.BASE_PATH);
+        }else if ("wmi".equals(configName)) {
```
Comment (patrick-schweizer) [inline features/config/mock/src/main/java/org/opennms/netmgt/dao/mock/ConfigurationManagerServiceMock.java:117]: nit: missing space?

File: opennms-config/src/test/java/org/opennms/netmgt/config/WmiPeerFactoryTest.java [hunk 2]
```diff
@@ -28,43 +28,61 @@
...
+    private WmiPeerFactory factory;
+
+    private void updateConfig(String amiConfigXml) {
+        try {
+            JaxbXmlConverter converter = new JaxbXmlConverter("wmi-config.xsd", "wmi-config", null);
+            String json = converter.xmlToJson(amiConfigXml);
+            factory.updateConfig(json);
+            factory.reload();
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (patrick-schweizer) [inline opennms-config/src/test/java/org/opennms/netmgt/config/WmiPeerFactoryTest.java:66]: rather do throw the exception since we are in a test?

**Review by patrick-schweizer — APPROVED** [review]
just 2 minor suggestions

File: opennms-config/src/test/java/org/opennms/netmgt/config/WmiPeerFactoryTest.java (hunk 2 above)
Comment (freddy61025) [reply opennms-config/src/test/java/org/opennms/netmgt/config/WmiPeerFactoryTest.java]: removed

File: features/config/mock/src/main/java/org/opennms/netmgt/dao/mock/ConfigurationManagerServiceMock.java (hunk 1 above)
Comment (freddy61025) [reply features/config/mock/src/main/java/org/opennms/netmgt/dao/mock/ConfigurationManagerServiceMock.java]: fixed

**Review by patrick-schweizer — APPROVED** [review]
:+1:

**Pull request merged**